openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -160,6 +138,75 @@ def _verify_profile_image_data(self, data, has_profile_image):
             }
         )
 
+
+@ddt.ddt
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
+@patch('openedx.core.djangoapps.user_api.accounts.image_helpers._PROFILE_IMAGE_SIZES', [50, 10])
+@patch.dict(
+    'openedx.core.djangoapps.user_api.accounts.image_helpers.PROFILE_IMAGE_SIZES_MAP',
+    {'full': 50, 'small': 10},
+    clear=True
+)
+@attr(shard=2)
+class TestAccountAPI(CacheIsolationTestCase, UserAPITestCase):
```

Comment:
How does `TestAccountAPI` differ from `TestAccountsAPI` below?  Should this class be renamed?